.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,34 +1,41 @@
 ---
```

Comment:
Should we add `*.md` to the ignored files by prettier?

Reply:
Yeah probably